webpack.config.cjs

```diff
@@ -229,7 +229,7 @@ const templateContent = ({ htmlWebpackPlugin }) => {
 const editorAndDashboard = {
   ...sharedConfig,
   entry: {
-    'edit-story': './assets/src/edit-story/index.js',
+    'edit-story': './packages/wp-story-editor/src/index.js', // @todo Change key name to wp-story-editor
```

Comment:
👍  to the todo

Reply:
I was trying to change the name but it leads to a series of changes, so will do it in a separate PR.